sdk/identity/identity/tests.yml

```diff
@@ -1,12 +1,56 @@
 trigger: none
 
+resources:
+  containers:
+    - container: 'ubuntu_netcore_keyring'
```

Comment:
Is this going to be added for additional languages as well? I think we should come up with a solution for sharing the container images, test secret variables and other resources in a language agnostic place, rather than having js/python/java/etc. tests take dependencies on configs, , and container publishing pipelines that exist in the azure-sdk-for-net repo.

Reply:
@benbp Praveen and I have been in conversations. We want to make sure we can make things work first, then we’re planning to reach out to the .NET team to discuss how to share these secret/resources. I’ll have an update later this week 👍

Reply:
Ok sounds good!